v1/common_schemas.yaml

```diff
@@ -108,20 +108,16 @@ components:
         lang:
           type: string
           description: The page language code
-          example: en
```

Comment:
Is there a reason for getting rid of the examples?

Reply:
ya. 'example' is not a supported key in json-schema, `examples` is. and in openapi `example` is the supported key, while `examples` is not. newer version of json validator package we use fails when it sees `example` and openapi-spec validator fails if it sees `examples`. So, I've decided to just remove them instead of messing with this incompatibility.

Reply:
Ohh ok. Makes sense